pkg/kube/apisix_route.go

```diff
@@ -54,21 +54,21 @@ type ApisixRouteInformer interface {
 // ApisixRoute is an encapsulation for ApisixRoute resource with different
 // versions, for now, they are apisix.apache.org/v1 and apisix.apache.org/v2alpha1
 type ApisixRoute interface {
-	// GroupVersion returns the api group version of the
+	// GetGroupVersion returns the api group version of the
```

Comment:
Frankly, these changes are trivial and useless, I suggest reverting them back, or it'll also pollute the result of git blame.

Reply:
@tokers  Thanks for your suggestion, I have another optimization idea, we could provide factory function that returns a concrete implementation runtime.Object for the given GVK instand of kube/apisix_route.go(or ingress.go etcd).

Reply:
That sounds good!

Reply:
> @tokers Thanks for your suggestion, I have another optimization idea, we could provide factory function that returns a concrete implementation runtime.Object for the given GVK instand of kube/apisix_route.go(or ingress.go etc).

@Belyenochi Will you update this PR based on this idea?

Reply:
> > @tokers Thanks for your suggestion, I have another optimization idea, we could provide factory function that returns a concrete implementation runtime.Object for the given GVK instand of kube/apisix_route.go(or ingress.go etc).
> 
> @Belyenochi Will you update this PR based on this idea?

tao, I'm trying to simplify this module, will update the commit in two weeks.

Reply:
Thanks!

Reply:
@Belyenochi do you have time to pick this up?